systemd/osdf-cache.yaml

```diff
@@ -0,0 +1,24 @@
+Debug: false
+Logging:
+  Level: "Error"
+Federation:
+  DiscoveryUrl: osg-htc.org
+  DirectorUrl: https://director-origins.osg-htc.org
+  TopologyNamespaceURL: https://topology.opensciencegrid.org/osdf/namespaces
+Server:
+  TLSCertificate: /etc/grid-security/xrd/xrdcert.pem
```

Comment:
Is there not a better system location?

Reply:
Not really. There's no 'standard' location for a host cert; Let's Encrypt puts the cert/key in `/etc/letsencrypt/live/$domain` for example. At least `/etc/grid-security/xrd/xrdcert.pem` is backward compatible with xcache-based cache/origin configurations; I can pick something else like `/etc/pelican/{cert,key}.pem` for the registry, director, and non-osdf configs.